### DISCUSSION TITLE
Unexpected dependency behaviour

### DISCUSSION THREAD

**Discussion by Woudjee** [NuGet/Home#13609] [General]
Hi again,

I have another behaviour that I expected differently after reading about it a lot. I'm talking about dependencies. Consider the following situation:

![image](https://github.com/NuGet/Home/assets/84737524/f6e4bde3-a948-4dac-b2ae-ac2613c9d834)

When adding my Nuget package in a different project, I would have expected to now see dependancies. However, I do not. What am I doing wrong?

![image](https://github.com/NuGet/Home/assets/84737524/0b07fc86-da08-443f-9e2b-2419df1509a4)

My interpretation of this is that what I have should work: https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files

Woudjee

**Comment by baronfel** [comment]
That should be correct - this is a pretty simple example that I would expect to Just Work.  Can you check that you're not looking at an old version of the nuget package you're creating? You may need to delete it from your bin directory to ensure it gets re-created.

**Comment by Woudjee** [comment]
Hi Baron, that is a good point. Unfortuntately I can confirm that what you're writing is not the problem. I have this exact same situation with all my Nuget packages (I have 5 now, some .NET Standard, some .NET Framework and this the only .NET Core). All my Nuget packages have this same issue.

**Reply by baronfel** [reply]
Ok, then it'll require an https://aka.ms/binlog to get to the details I'm afraid. You can get one with `dotnet pack -bl`.

**Reply by Woudjee** [reply]
Alright. I believe this is what you're looking for. I have put it into a ZIP-file.
[msbuild.zip](https://github.com/user-attachments/files/16131210/msbuild.zip)

**Reply by Woudjee** [reply]
Have you had the opportunity to look into it?
